Jellyfin.ApiClient/ApiWebSocket.cs

```diff
@@ -108,17 +103,17 @@ private async Task EnsureConnectionAsync(CancellationToken cancellationToken)
                 {
                     var socket = _webSocketFactory();
 
-                    Logger.Info("Created new web socket of type {0}", socket.GetType().Name);
+                    Logger.LogInformation("Created new web socket of type {0}", socket.GetType().Name);
 
-                    Logger.Info("Connecting to {0}", url);
+                    Logger.LogInformation("Connecting to {0}", url);
 
                     await socket.ConnectAsync(url, cancellationToken).ConfigureAwait(false);
 
-                    Logger.Info("Connected to {0}", url);
+                    Logger.LogInformation("Connected to {0}", url);
 
                     socket.OnReceiveBytes = OnMessageReceived;
                     socket.OnReceive = OnMessageReceived;
-                    socket.Closed += _currentWebSocket_Closed;
+                    socket.Closed += CurrentWebSocket_Closed;
```

Comment:
The style we have would use OnCurrentWebSocketClosed here.

Reply:
Done in 3641a54..